Services/DataBaseService/Startup.cs

```diff
@@ -3,15 +3,26 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Configuration;
+using DataBaseService.Utils;
 
 namespace DataBaseService
 {
     public class Startup
     {
+        static IConfiguration Configuration { get; set; }
```

Comment:
public, private, internal, protected, internal protected, private protected?

Reply:
Public and nonstatic. Don't remember, why it was marked such way at all